superbit_lensing/metacalibration/ngmix_fit_superbit3.py

```diff
@@ -174,31 +219,41 @@ def _get_jacobians(self, source_id=None):
     def _get_source_observations(self,source_id = None,psf_noise = 1e-6):
         jaclist = self._get_jacobians(source_id)
         psf_cutouts = self.medsObj.get_cutout_list(source_id, type='psf')
+        #psf_cutouts = self._make_psfex_cutouts(source_id)
         weight_cutouts = self.medsObj.get_cutout_list(source_id, type='weight')
         image_cutouts = self.medsObj.get_cutout_list(source_id, type='image')
         image_obslist = ngmix.observation.ObsList()
 
         for i in range(len(image_cutouts)):
 
+            jj = jaclist[i]
+
+            try:
+                xcenter = psf_cutouts[i].true_center.x
+                ycenter = psf_cutouts[i].true_center.y
+                jj_psf = ngmix.DiagonalJacobian(scale=psf_cutouts[i].scale,x=xcenter,y=ycenter)
+            except AttributeError:
+                jj_psf = jj
+
             # Apparently it likes to add noise to the psf.
             this_psf = psf_cutouts[i] + psf_noise * np.random.randn(psf_cutouts[i].shape[0],psf_cutouts[i].shape[1])
+            #this_psf = psf_cutouts[i].array + psf_noise * np.random.randn(psf_cutouts[i].array.shape[0],psf_cutouts[i].array.shape[1])
             this_psf_weight = np.zeros_like(this_psf) + 1./psf_noise**2
 
-            this_image = image_cutouts[i]
-
             # Treat sky background variance as a Poisson distribution, e.g.
             #     - mean bkg = 0.048*600
             #     - std_dev = sqrt(bkg) = 5.3
             #     - sky_sigma = std_dev**2 = 25.1
 
-            sky_sigma = (2)**2
+            sky_sigma = (4.7)**2
```

Comment:
where does this value come from?

Reply:
The sky background RMS of the simulated imaging data.

Reply:
This value (4.7)^2 is not the same as the value used earlier (0.0557*300)^2, which equals (16.71)^2. This is what worries me about having it hardcoded. Another bump for the new issue

Reply:
Agreed. I will mark this conversation as resolved, but we should make this a separate issue.

Reply:
See #37